code/src/reducers/todos.js

```diff
@@ -0,0 +1,86 @@
+import { createSlice } from '@reduxjs/toolkit'
```

Comment:
Hi Sara!
I really enjoyed having a look at your clean, organized code and I have learned a lot as well! The only point can be getting rid off comments provided by the teachers! Good job!